### PR TITLE
Add parallelisation section to docs

### LINKED ISSUE #195: Documentation improvement
Minor things to improve in the docs:

- Parallelisation: mention that Slurm is not required for locally running an external model in parallel.
- Improve model for OpenSees.

### PR DESCRIPTION
**Pull request by FriesischScott**
I added a section on parallelisation and updated the OpenSees example. I also fixed the outdated info in the hpc section and fixed a few errors/warnings here and there.

The response surface docs need more work in the future.

Closes #195,#220.

### CODE REVIEW

File: docs/src/manual/parallelisation.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+end
+
+samples = sample([x, y], 1000)
+evaluate!(m, samples)
+
+rmprocs(workers()) # release the local workers
+```
+
+It is important to note, that the [`ParallelModel`](@ref) requires some overhead to distribute the function calls to the workers. Therefore it performs significantly slower than the standard [`Model`](@ref) with vectorized operations for a simple function as in this example. The method of executing a model in parallel presented above also applies to the [`ExternalModel](@ref).
```
Comment (AnderGray) [inline docs/src/manual/parallelisation.md:33]: Missing an ` :

`['ExternalModel']`

File: docs/literate/external/01-opensees.jl [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#===
+# External Solvers
+
+## OpenSees supported beam
+
+In this example we will perform the reliability analysis of a supported beam using the open-source finite element software [OpenSees](https://opensees.berkeley.edu/).
+The example definition can be found [here](https://opensees.berkeley.edu/wiki/index.php/Simply_supported_beam_modeled_with_two_dimensional_solid_elements)
+We will be modeling the Young's modulus of the ElasticIsotropic in the OpenSees model as a  [`RandomVariable`](@ref).
```
Comment (AnderGray) [inline docs/literate/external/01-opensees.jl:8]: Should `ElasticIsotropic ` be `isotropic elasticity`?

File: docs/literate/external/01-opensees.jl [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+# These files will be rendere through Mustache.jl and have values injected
+sourcefile = "supported-beam.tcl"
+
+# Dictionary to map format Strings (Format.jl) to variables
+numberformats = Dict(:E => ".8e")
+
+# UQ will create subfolders in here to run the solver and store the results
+workdir = joinpath(pwd(), "supported-beam")
+
+# Read output file and compute maximum (absolute) displacement
```
Comment (AnderGray) [inline docs/literate/external/01-opensees.jl:28]: Missing full stop after `displacement.`

File: docs/literate/external/01-opensees.jl [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+# An extractor is passed the working directory for the current sample as `base`
```
Comment (AnderGray) [inline docs/literate/external/01-opensees.jl:29]: is passed `to` the working directory

File: docs/literate/external/02-opensees-parallel.jl [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+# load the Distributed module
+using Distributed
+
+# add six local workers
+addprocs(6)
+
+# The setup of the example is enclosed in the @everywhere macro. This ensures, that the main process and all worker processes have loaded the necessary modules and received all defined variables and functions.
+
+@everywhere begin
```
Comment (AnderGray) [inline docs/literate/external/02-opensees-parallel.jl:15]: Are you missing the `end` to this `begin`?

File: docs/literate/external/02-opensees-parallel.jl [hunk 6]
```diff
@@ -0,0 +1,66 @@
+
```
Comment (AnderGray) [inline docs/literate/external/02-opensees-parallel.jl:1]: It seems that this example is a pretty much replica of the non-parallel version. Is it possible to compress some of the text. I'm not sure we need to say everything again.

I think after you setup the parallel part, you can pretty much say something like `after this, the setup is the same as the serial evaluation`, and just show the code without the elaboration.

**Review by AnderGray — CHANGES_REQUESTED** [review]
Nice update. Some minor comments and typos I've found.

File: docs/literate/external/01-opensees.jl (hunk 2 above)
Comment (FriesischScott) [reply docs/literate/external/01-opensees.jl:8]: That's the name of the material in OpenSees. But I it is missing the word *material*.

File: docs/literate/external/01-opensees.jl (hunk 4 above)
Comment (FriesischScott) [reply docs/literate/external/01-opensees.jl:29]: What I mean is that the variable `base` is passed to the extractor as  the input argument. I'll reword it.

File: docs/literate/external/02-opensees-parallel.jl (hunk 5 above)
Comment (FriesischScott) [reply docs/literate/external/02-opensees-parallel.jl:15]: No, it's just very far down.

File: docs/literate/external/02-opensees-parallel.jl (hunk 6 above)
Comment (FriesischScott) [reply docs/literate/external/02-opensees-parallel.jl:1]: True. I removed the comments and referred to the serial example.

**Review by AnderGray — APPROVED** [review]

**Pull request merged**